src/test/java/reposense/system/CommandRunnerTest.java

```diff
@@ -180,6 +181,46 @@ public void getCommitHashBeforeDate_invalidBranch_throwsRunTimeException() {
         CommandRunner.getCommitHashBeforeDate(config.getRepoRoot(), "invalidBranch", date);
     }
 
+    @Test
+    public void removeCommitAuthor_singleValidCommitHash_success() {
+        // create new temp branch so as to not affect the master branch for other tests
+        CommandRunner.checkoutNewBranch(config.getRepoRoot(), TEMP_BRANCH);
```

Comment:
why don't just use the `removeAuthorFromIgnoredCommits` method?

Reply:
its a unit test within `CommandRunner`

Reply:
ic. unit test is good but testing the `removeAuthorFromIgnoredCommits` is more important.
do include it in the next iteration.